client/src/components/NavBar/NavBar.tsx

```diff
@@ -59,7 +59,7 @@ const NavBar = ({ loggedInUser, handleDrawerToggle }: Props): JSX.Element => {
       <AppBar position="static">
         <Toolbar>
           <Typography variant="h6" className={classes.title}>
-            {activeBoard.name}
+            {activeBoard._id != 'Initial' && activeBoard.name}
```

Comment:
Confused with the idea here;  doesn't the `activeBoard` start with a name of `Initial` rather than `_id`?

Reply:
@amahalwy , since we don't have the possibility of activeBoard being null, I check if I am dealing with the initial dummy board. If not, then I render its name. I chose to compare it to the id because the user might name a board 'Initial' and then we would have a bug. This should all be taken care of when we can handle null board objects.